pdns/README-dnsdist.md

```diff
@@ -1460,6 +1460,7 @@ instantiate a server with additional parameters
         * `truncate(bits)`: truncate the address to the specified number of bits
     * DNSName related:
         * `newDNSName(name)`: make a DNSName based on this .-terminated name
+        * member `chopOff()`: remove left-most label
```

Comment:
Please document the return value. I find myself using that a lot when writing Lua for the recursor.